modules/light-clients/08-wasm/types/client_state.go

```diff
@@ -121,7 +121,7 @@ func (cs ClientState) Initialize(ctx sdk.Context, cdc codec.BinaryCodec, clientS
 		ConsensusState: consensusState,
 	}
 
-	return wasmInstantiate(ctx, clientStore, &cs, payload)
+	return wasmInstantiate(ctx, cdc, clientStore, &cs, payload)
```

Comment:
we could also move the `payload` arg to be more consistent with `wasmSudo` or vice versa. I like the payload being the last arg, but also fine with whatever